content/docs/language.md

```diff
@@ -1643,11 +1641,11 @@ Eval = State (Map string i32)
 
 lookup (name: string) -> Maybe i32 with Eval =
     map = get ()
-    map.get name
+    map |> get name
```

Comment:
Lets revert this change, my design for `.` isn't quite complete yet (hence the lack of actual documentation on it) but it is meant to have subtley different name resolution implications than the pipe operator. Namely, it should look in the module where the type of its lhs was declared for the name first to avoid extra importing. Without this there would be two `get`s in scope here.